src/TodoList.jsx

```diff
@@ -0,0 +1,37 @@
+import * as React from 'react';
+
+const todoList = [
+    {
+        id: 1,
+        title: "Commit project and create working branch"
+    },
+    {
+        id: 2,
+        title: "Create the beginnings of the todo list"
+    },
+    {
+        id: 3,
+        title: "Complete assignment"
+    }
+];
+
+function listItem(item) {
+    return (
+        <li key={item.id}>
+            <span>{item.id} </span>
+            <span>{item.title}</span>
+        </li>
+    )
+}
+
+function TodoList() {
+    return (
+        <ul>
+            {todoList.map(listItem)}
```

Comment:
Building on my comments from the last assignment - would actually recommend for now that you refactor this code to bring in the `<li>` into this return statement instead of abstracting out to a separate function. So make this `todoList.map((item) => (<li>...</li>)`

I see what you're trying to do, but I would save this abstraction for future lessons when we go through how to appropriately create React components with custom properties (which is how you'd want to do this abstraction)